wafer/schedule/models.py

```diff
@@ -68,31 +111,41 @@ class Slot(models.Model):
                                                   "slot OR a day and start "
                                                   "time set)"))
 
-    day = models.ForeignKey(Day, null=True, blank=True,
+    block = models.ForeignKey(ScheduleBlock, null=True, blank=True,
```

Comment:
Now that slots have datetimes, I'm wondering whether it makes sense to drop the link from slots to blocks and simply use the blocks as time range and venue filters for displaying slots with?

Reply:
Noting that this was done in e2acebf